pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -179,14 +195,6 @@ func (ghs *githubScraper) getOldestBranchCommit(repo *Repo, branch *Branch) {
 func (ghs *githubScraper) getRepoPullRequestInformation(repo *Repo) {
```

Comment:
I really thing we should pass `*githubv4.Client` to the functions and break this out a little bit

Reply:
follow up issue: https://github.com/liatrio/liatrio-otel-collector/issues/49